frontend/src/app/patients/Components/AddPatientsHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import { LinkWithQuery } from "../../commonComponents/LinkWithQuery";
+import {
+  PATIENT_TERM,
+  PATIENT_TERM_PLURAL_CAP,
+} from "../../../config/constants";
+import PatientsNav from "../PatientsNav";
+import iconSprite from "../../../../node_modules/uswds/dist/img/sprite.svg";
+import { RootState } from "../../store";
+
+interface Props {
+  additional?: React.ReactElement;
```

Comment:
Same question than before. Does the property really need to be an element or would a node could fit the bill? for example if it only will be traditional DOM